### PR TITLE
Des 849

### PR DESCRIPTION
**Pull request by wyseguyonline**

### CODE REVIEW

**Comment by wyseguyonline** [comment]
This includes changes to both the PhoneNumber, Dropdown, and DropdownItem components.

**Comment by StommePoes** [comment]
Is this at a point where I should try a full usability/a11y test? Or are some more fixes in the pipeline soon?

File: demo/demoPages/components/DropdownPage.js [hunk 1]
```diff
@@ -45,16 +48,19 @@ class DropdownPage extends React.Component {
               <li className="li-props">checkmark: whether or not to leave spacing for a selected checkmark</li>
               <li className="li-props">selected: accessibility/globalization word for 'selected'</li>
               <li className="li-props">selectedName: accessibility label for name of item</li>
+              <li className="li-props">imgUrl: URL to image (if type is imgButton)</li>
```
Comment (mhomolak) [inline demo/demoPages/components/DropdownPage.js]: Just to avoid any confusion maybe have imgButton be imageButton for the actual prop name.

**Comment by wyseguyonline** [comment]
@StommePoes At this time, we should limit our A11y discussion to the PhoneNumber component and create a ticket to address any A11y issues that exist in the dropdown.  I'd like to limit scope creep.  So if there are A11y issues related to the PhoneNumber component specifically, we should fix those.  A11y issues related to dropdown should be cataloged and entered into a separate ticket.

**Comment by umahaea** [comment]
all approved. merging it.

**Pull request merged**